index.html

```diff
@@ -41,6 +41,132 @@ <h2>LET'S CONNECT</h2>
+          <div class="button">
+            <button class="button1" type="button">See Project</button>
+          </div>
+        </div>
+      </section>
+
+      <section class="project project2">
+        <img src="./images/portfolio-2.png" alt="My first project">
+        <div class="description">
+          <h3 class="title-project">Multi-Post Stories</h3>
+          <ul class="dates">
+            <li class="item1">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2021</li>
+          </ul>
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="button">
+            <button class="button1" type="button">See Project</button>
+          </div>
+        </div>
+      </section>
+
+      <section class="project project3">
+        <img src="./images/portfolio-3.png" alt="My first project">
+        <div class="description">
+          <h3 class="title-project">Tonic</h3>
+          <ul class="dates">
+            <li class="item1">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2021</li>
+          </ul>
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="button">
+            <button class="button1" type="button">See Project</button>
+          </div>
+        </div>
+      </section>
+
+      <section class="project project4">
+        <img src="./images/portfolio-4.png" alt="My first project">
+        <div class="description">
+          <h3 class="title-project">Multi-Post Stories</h3>
+          <ul class="dates">
+            <li class="item1">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2021</li>
+          </ul>
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="button">
+            <button class="button1" type="button">See Project</button>
+          </div>
+        </div>
+      </section>
+    </div>
+
+    <div class="wrapper2">
+      <div class="second">
+        <h4>About<br>
+          Myself
+        </h4>
+        <p>
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2>LET'S CONNECT</h2>
```

Comment:
- [ ] I think you are confusing the heading hierarchy here. I believe `<h2>LET'S CONNECT</h2>` is meant to be wrapped in an `<h4>` tag instead while the ` <h4>About<br>Myself</h4>` takes the `<h2>` tag. 

❓ What do you think is the more important tag 🤷🏻